### ISSUE TITLE
[BUG] [Linter] H017 false positive

### ISSUE THREAD

**Issue by monosans** [Riverside-Healthcare/djLint#406]
- [X] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [X] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [X] I've read the [docs](https://djlint.com)

## System Info
 - OS: Arch Linux
 - Python 3.10.7
 - djLint 1.17.2
 - Template language: django

## Issue

H017 false positive

## How To Reproduce

```django
<input type="number"
       {% if item.quantity > 0 %}
         min="1" value="1" max="{{ item.quantity }}"
       {% else %}
         min="0" value="0" max="0"
       {% endif %}
       />
```
```django
H017 1:0 Tag should be self closing. <input type="number"
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]

The same bug appears with the default config. However, I'll leave my config just in case.

```toml
[tool.djlint]
indent = 2
profile = "django"
max_line_length = 80
format_attribute_template_tags = true
```

**Comment by christopherpickering** [comment]
Thanks! Reducing to the simplest error example:
```html
<input {{ > }} />
```
